itou/siae_evaluations/models.py

```diff
@@ -334,7 +333,17 @@ def close(self):
                     if len(criterias) == 0 or any(crit.submitted_at is None for crit in criterias):
                         siae_without_proofs.append(evaluated_siae)
                         has_siae_to_notify = True
-            emails = [SIAEEmailFactory(evaluated_siae).refused_no_proofs() for evaluated_siae in siae_without_proofs]
+                elif evaluated_siae.state == evaluation_enums.EvaluatedSiaeState.NOTIFICATION_PENDING:
+                    emails.append(SIAEEmailFactory(evaluated_siae).refused())
+                elif evaluated_siae.state == evaluation_enums.EvaluatedSiaeState.ACCEPTED:
+                    if evaluated_siae.reviewed_at != evaluated_siae.final_reviewed_at:
+                        # This check ensures that the acceptance happened in the adversarial stage
+                        # and not the amicable one
+                        emails.append(SIAEEmailFactory(evaluated_siae).reviewed(adversarial=True))
```

Comment:
Je pense que les SIAE qui ont submit en phase contradictoire (état `SUBMITTED`), mais n’ont pas été revue par la DDETS vont aussi recevoir cet email, qui est faux par rapport à ce qui s’est passé, la DDETS n’a pas accepté.

Reply:
En fait non: dans ce `elif` on est après `if evaluated_siae.final_reviewed_at is None:` donc `final_reviewed_at` vaut forcément quelque chose et la DDETS a donc bien validé.

Reply:
Même pas: comme le `ended_at` est mis à jour au début de la fonction, l'état `SUBMITTED` n'est plus possible (il se transforme en `ACCEPTED` ) :weary:

Reply:
Pas besoin d’être à l’état `SUBMITTED` pour de vrai (c’est un abus de langage de ma part, j’aurais du dire : qui ont envoyé des docs et n’ont pas été revues par la DDETS avant la fin de la campagne). Dans ce cas, la SIAE rentre dans `if evaluated_siae.final_reviewed_at is None:`, donc on ne lui envoie pas de mail. Effectivement, son état est `ACCEPTED`, mais comme tu l’as remarqué, on n’entre pas dans le `elif evaluated_siae.state == ACCEPTED`.